src/backend/optimizer/prep/prepunion.c

```diff
@@ -1808,6 +1808,44 @@ adjust_appendrel_attrs(PlannerInfo *root, Node *node, AppendRelInfo *appinfo)
 	return result;
 }
 
+static bool
+nested_subplan_mutator(Node *node, plan_tree_base_prefix *context) 
+{
+	if (node == NULL) 
+	{
```

Comment:
Minor. We can ignore brackets here and save two lines :)

Reply:
This wal also fixed